Makefile

```diff
@@ -625,17 +625,14 @@ $(OBJDIR)/ceed.pc : pkgconfig-prefix = $(prefix)
 	    -e "s:%libs_private%:$(pkgconfig-libs-private):" $< > $@
 
 ifeq ($(filter install,$(MAKECMDGOALS)),install)
-  CPPFLAGS += $(if $(ASAN),$(AFLAGS)) -DCEED_JIT_SOUCE_ROOT_DEFAULT="\"$(abspath $(DESTDIR)$(includedir))/\""
+  $(OBJDIR)/interface/ceed-jit-source-root-default.o : CPPFLAGS += -DCEED_JIT_SOUCE_ROOT_DEFAULT="\"$(abspath $(includedir))/\""
```

Comment:
One should use `prefix=` or `includedir=` to change the install location. `DESTDIR` is used a staging area for package managers, but it isn't intended that you run from there.